spec/units/listing_spec.rb

```diff
@@ -1,19 +1,33 @@
 require 'listing.rb'
 
 describe Listing do
-  
+
   describe 'self.create' do
     it 'adds a listing to the database' do
-      pg_instance_returned = Listing.create('testtitle')
-      expect(pg_instance_returned[0]['title']).to eq 'testtitle'
+      conn = PG.connect(dbname: 'abodenb_test')
+      Listing.create(title: 'testtitle', description: 'test description', price: '100')
+      result = conn.exec("SELECT title FROM listings;")
+      expect(result[0]['title']).to eq 'testtitle'
     end
   end
 
   describe 'self.all' do
-    it 'returns an array of all listings in the database' do
+    it 'returns an array of listing objects of all listings in the database' do
       conn = PG.connect(dbname: 'abodenb_test')
       conn.exec("INSERT INTO listings (title) VALUES ('testtitle');")
-      expect(Listing.all).to eq ['testtitle']
+      conn.exec("INSERT INTO listings (title) VALUES ('testtitle2');")
+      expect(Listing.all.first).to be_kind_of(Listing)
+      expect(Listing.all.last).to be_kind_of(Listing)
+      expect(Listing.all.length).to eq 2
+      expect(Listing.all.first.title).to eq 'testtitle'
+      expect(Listing.all.last.title).to eq 'testtitle2'
```

Comment:
maybe same point as above?

Reply:
We were trying to keep this test a little isolated: test specifically that what it returns are listings, that it's an array of listings, there's the right amount of listings, and they're two different listings 

The other tests are to test that the right info is being added into the database - so think adding price/description into this would just be bloating a bit?